labs/SParts.md

```diff
@@ -0,0 +1,25 @@
+# Lab Name
+
+SParts Project
```

Comment:
Can we change any usage of project to lab in this proposal? We want to be sure that Hyperledger labs are separate from Hyperledger projects.